data-access/README.md

```diff
@@ -17,8 +17,6 @@
   under the License.
 -->
 
-# DataFusion Storage
+# Data Access Layer
 
-This module contains an `async` API for the [DataFusion][df] to access data, either remotely or locally.
-
-[df]: https://crates.io/crates/datafusion
+This module contains an `async` API for accessing data based on object store interfaces, either remotely or locally.
```

Comment:
If i understand the intent of this correctly, i think the data access layer could expose other data access methods than just object store (although they may not exist yet). For example a streaming provider, or database provider.  If this is correct i think we would just want to make the description more generic.

Reply:
Agree. Although currently we only support object store, we can make description not based on the object store.